### DISCUSSION TITLE
Why does DatePicker always include time portion now?

### DISCUSSION THREAD

**Discussion by madsem** [filamentphp/filament#3285] [Help]
The DatePicker (I think), was not always including the time portion before.

Now when trying to get rid of the time, it just always includes it, even when doing:
```php
Filter::make('date')
                ->form([
                    Grid::make()
                        ->schema([
                            Forms\Components\DatePicker::make('date_from')
                                ->default(now()->toDateString())
                                ->withoutTime(),
                            Forms\Components\DatePicker::make('date_until')
                                ->default(now()->toDateString())
                                ->withoutTime(),
                        ]),
                ]),
```

The filters in URL still look like this:
`http://localhost/accounts?tableFilters[date][date_from]=2022-07-29+09:01:52&tableFilters[date][date_until]=2022-07-29+09:01:52`

How can you disable appending the time?

**Comment by madsem** [comment]
I believe this is has something to do with the new session filters, when I set it to false:
```php
protected function shouldPersistTableFiltersInSession(): bool
    {
        return false;
    }
```

The time portion of the filters are set to 00:00: `http://localhost/accounts?tableFilters[date][date_from]=2022-07-29+00:00:00&tableFilters[date][date_until]=2022-07-29+00:00:00`

That at least makes it possible to query for today again, which was failing before.

**Reply by zepfietje** [reply]
@madsem, if you believe it's a bug, open an issue for it and we'll have a look!
